client/src/utils/__fixtures/curriculum-helpers-javascript.js

```diff
@@ -12,10 +12,7 @@ nonMutatingPush(first, second);`;
 
 const jsCodeWithSingleAndMultLineCommentsRemoved = `
 function nonMutatingPush(original, newItem) {
-
-
-
-  return original.push(newItem);
+    return original.push(newItem);
```

Comment:
Why is this indented 4 spaces?

Reply:
Because the removed block comment has two spaces in front of it (see [post](https://github.com/freeCodeCamp/freeCodeCamp/issues/41951#issuecomment-830661282) in issue).